scripts/pkg/build_templates/opensearch/deb/debian/postinst

```diff
@@ -37,11 +37,17 @@ if ! grep -q '## OpenSearch Performance Analyzer' ${config_dir}/jvm.options; the
    echo "--add-opens=jdk.attach/sun.tools.attach=ALL-UNNAMED" >> ${config_dir}/jvm.options
 fi
 
-# Set owner
-chown -R opensearch.opensearch ${product_dir}
+# Set ownership and permissions
```

Comment:
Need to make some changes here as well:
 https://github.com/opensearch-project/opensearch-build/blob/main/scripts/pkg/build_templates/opensearch/deb/debmake_opensearch_install.sh#L44

Reply:
This is supposed to be noop, but for consistency I added a commit to express the permissions explicitly rather than adding and removing permissions from a default that is implied.

Reply:
Got it, thanks.